pkg/metrics/otel.go

```diff
@@ -134,6 +134,26 @@ processors:
           # included by the 'regex include' filter above and is not included in
           # our Monarch metric definitions
           - kcc_resource_count_total
+  attributes/kubernetes:
+    actions:
+      - key: configsync.sync.kind
+        action: delete
+      - key: configsync.sync.name
+        action: delete
+      - key: configsync.sync.namespace
+        action: delete
+      - key: k8s.pod.uid
+        action: delete
+      - key: k8s.pod.ip
+        action: delete
+      - key: k8s.node.name
+        action: delete
+      - key: k8s.deployment.name
+        action: delete
+  memory_limiter:
```

Comment:
does memory_limiter need to be added in this PR? Or can it be added separately, in case in needs to be reverted separately?

Reply:
+1 on Karl's comment.

Reply:
Agreed, removed from this change.